frontend/src/components/ItemMetadataForm.js

```diff
@@ -19,9 +36,11 @@ const ItemMetadataForm = ({
     helperText={ errors.title ? errors.title.join(' ') : null }
     disabled={ disabled }
     label='Title'
+    placeholder='Enter a title, eg. Boundary Singularities'
```

Comment:
``placeholder='Boundary Singularities'``

"Enter a title" is unnecessary since the control is labelled "title".